### DISCUSSION TITLE
For a PDF which uses the margin, how do I make a full width titlepage?

### DISCUSSION THREAD

**Discussion by sasja-san** [quarto-dev/quarto-cli#8249] [Q&A]
### Description

I'm trying to make a really simple title page, on its onw separate page.  If I use `\maketitle` then the content of the title appears twice, which hardly seems like what I want. I added
```
format:
    pdf:
      template-partials:
        - title.tex
```
and tried to do some stuff in that document but I couldn't succeed at all. Any advice on this is welcome.
It seems like the `geometry` package forces everything to be its way, and I have no clue how I override it.

```qmd
---
title: Two lines, but not when using the margin.
subtitle: Hopefully this subtitle is long enough to span two/three lines. And is there some way to force a newline in a title or subtitle?
date: today
date-format: YYYY-MM-DD
author: Jane-John Doe, <send.newdz@hotmale.com>

format:
    pdf:
      papersize: a4
      documentclass: scrartcl
      geometry:
        - showframe # Debugging purposes 
        - left=15.0mm
        - textwidth=102.0mm
        - marginparsep=7.7mm
        - marginparwidth=70.3mm
        - top=20mm
      include-in-header:
        - text: |
            \usepackage{lipsum}
---

\titlepage

# Here is a new chapter


\lipsum[100]

# And another

\lipsum[20]

```

Ideal would be if I just could do

```qmd
:::{.column-page}
\titlepage
:::
```
and that just worked. Then I could add my stupid blue line and my examiners name in `title.tex` and be happy.

**Comment by mcanouil** [comment]
That's a LaTeX question of which I don't have the answer now but it's definitely on the Internet.
Since you are already using raw LaTeX, you could apply more LaTeX.

As for HTML advanced customisation you need to learn/use HTML/CSS/JS, here you need to learn/use LaTeX and understand the partial/template to make your own modifications.

Note that you should not use raw code outside of raw code block which is in your case ` ```{=latex} `.

Also when working with LaTeX based documents, you can use `keep-tex: true` to have a look at the LaTeX document you are producing.

Finally, you might be interested in the "titlepage" Quarto extension: <https://github.com/nmfs-opensci/quarto_titlepages>.

**Comment by sasja-san** [comment]
Hmm... apparently a shitty naming convention is in the works here...

> Ignore the seductively-named title.tex because to edit the title page you will need to replace before-body.tex

from https://cameronpatrick.com/post/2023/07/quarto-thesis-formatting/

**Reply by mcanouil** [reply]
It's described ...

https://quarto.org/docs/journals/templates.html#latex-partials

![image](https://github.com/quarto-dev/quarto-cli/assets/8896044/d05a8e85-a59f-4807-9d69-dc24c807f935)

**Reply by sasja-san** [reply]
Sure. But it doesn't make the naming convention any less shitty.

**Reply by mcanouil** [reply]
That's not very constructive or helpful ...
"Title" refers to "title block" that's the rational.

Note that calling things "shitty" and making this kind of comments is against the code of conduct: https://github.com/quarto-dev/quarto-cli#contributor-covenant-code-of-conduct
If you have a feedback, please share it respectfully and constructively. Thank you.
